src/components/views/messages/MessageActionBar.js

```diff
@@ -149,45 +197,61 @@ export default class MessageActionBar extends React.PureComponent {
 
     render() {
         const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
-
-        let reactButton;
-        let replyButton;
-        let editButton;
+        const buttons = [];
 
         if (isContentActionable(this.props.mxEvent)) {
             if (this.context.canReact) {
-                reactButton = (
-                    <ReactButton mxEvent={this.props.mxEvent} reactions={this.props.reactions} onFocusChange={this.onFocusChange} />
+                buttons.push(
+                    <ReactButton mxEvent={this.props.mxEvent} reactions={this.props.reactions} onFocusChange={this.onFocusChange}
```

Comment:
can you style these props like the reply button below (one per row starting on row after opening

Reply:
Sure